repos/system_upgrade/common/libraries/dnfplugin.py

```diff
@@ -65,6 +65,15 @@ def install(target_basedir):
         )
 
 
+def rebuild_rpm_db(context, root=None):
```

Comment:
if we do not expect to call this function directly from actors, let's make it private:
```suggestion
def _rebuild_rpm_db(context, root=None):
```

Reply:
Makes sense + we also have to call this function only when upgrading to el9. Now, it would fail during el7 > el8.